### PR TITLE
reformat + acks section

### PR DESCRIPTION
**Pull request by bumblefudge**
- adds acknowledgements policy discussed at [July Technical Steering Committee meeting](https://github.com/decentralized-identity/TSC/blob/main/agenda.md#meeting---7-july-2021---1300-et) 
- cleans up line-breaks and a few minor editorial nits

### CODE REVIEW

**Review by peacekeeper — APPROVED** [review]

**Review by OR13 — CHANGES_REQUESTED** [review]
> If the WG decides by consensus not to include an acknowledgments section, this decision should be recorded in the minutes/issues for the sake of transparency.

I don't believe this should be allowed... especially with no formal definition of `consensus`.

File: work-item-lifecycle.md [hunk 1]
```diff
@@ -6,43 +6,105 @@
...
-Once a Working Group believes it has achieved the objectives for its deliverable as described in the Scope, it will progress its Draft Deliverable to “Working Group Approved” status.
-1. Where protocols or interfaces are being specified, a section explicitly defining **test vectors for conformance testing** SHALL be written at time of approving a draft or earlier.
-2. Approved drafts should be put at a **permanent link** tracked on DIF's website or github.
+Once a Working Group believes it has achieved the objectives for its deliverable
+as described in the Scope, it will progress its Draft Deliverable to “Working
+Group Approved” status.
+1. Where protocols or interfaces are being specified, a section explicitly
+   defining **test vectors for conformance testing** SHALL be written at time of
+   approving a draft or earlier.
+2. The WG approval process SHOULD consider adding a "Acknowledgments" section to
```
Comment (OR13) [inline work-item-lifecycle.md:57]:
```suggestion
2. The WG approval process MUST consider adding a "Acknowledgments" section to
```

File: work-item-lifecycle.md [hunk 2]
```diff
@@ -6,43 +6,105 @@
...
+   the specification, particularly if the individuals and organization listed as
+   editors and contributors would like to add information about funding sources
+   or institutional support such as research grants or university labs. If the
```
Comment (OR13) [inline work-item-lifecycle.md:60]:
```suggestion
   or institutional support such as research grants or university labs. 
```

File: work-item-lifecycle.md [hunk 3]
```diff
@@ -6,43 +6,105 @@
...
+   WG decides by consensus not to include an acknowledgments section, this
```
Comment (OR13) [inline work-item-lifecycle.md:61]:
```suggestion
   
```

File: work-item-lifecycle.md [hunk 4]
```diff
@@ -6,43 +6,105 @@
...
+   decision should be recorded in the minutes/issues for the sake of
```
Comment (OR13) [inline work-item-lifecycle.md:62]:
```suggestion
   
```

File: work-item-lifecycle.md [hunk 5]
```diff
@@ -6,43 +6,105 @@
...
+   transparency.
```
Comment (OR13) [inline work-item-lifecycle.md:63]:
```suggestion
 
```

**Review by OR13 — CHANGES_REQUESTED** [review]
Specifically, working groups MUST NOT be allowed to obstruct acknowledgements from contributors, when requested.

**Comment by OR13** [comment]
Related PR which should be merged before this PR is:

https://github.com/decentralized-identity/sidetree/pull/1133

**Comment by OR13** [comment]
This PR is now blocked by the requested changes here: https://github.com/decentralized-identity/sidetree/pull/1133#discussion_r783318836

**Comment by OR13** [comment]
@csuwildcat you requested the following structure for acknowledgements here: https://github.com/decentralized-identity/sidetree/pull/1133#discussion_r783318836

> I would suggest changing the language to: "ENTITY_NAME_HERE received funding from the United States Department of Homeland Security's (US DHS) Silicon Valley Innovation Program to contribute to this work item under contracts 70RSAT20T00000003, and 70RSAT20T00000033. This work item does not necessarily reflect the position or the policy of the U.S. Government and no official endorsement should be inferred."

I would request that request this format of all work item acknowledgement section on this pull request.

We will assume you will be successful, and update our PR accordingly, but this PR should not be merged until proper guidance is given.

File: work-item-lifecycle.md (hunk 1 above)
Comment (bumblefudge) [reply work-item-lifecycle.md:57]: Ran this by the TSC at today's meeting, there was not support for this being mandatory. if no one PRs in any acknowledgements, should there be an empty section?

Comment (bumblefudge) [reply work-item-lifecycle.md:57]: Maybe a better wording than "MUST consider" is "MUST make a last call for acknowledgement PRs" to make it clear that WG members are responsible to PR in their acknowledgments before or as part of final editorial pass.

**Comment by bumblefudge** [comment]
Hey @OR13 , updated the PR a bit and discussed at yesterday's TSC.  I think TSC, not SC, has the final say on `work-item-lifecycle.md` and hopefully the addition of a template makes future cases more cut-and-dry.  Apologies for all the delay. Can I ask you to re-review?

**Pull request merged**